.github/workflows/pr_tests.yaml

```diff
@@ -33,11 +33,15 @@ jobs:
       - name: Install dependencies 📦
         run: |
           npm install
-          npx lerna bootstrap
+
+          for package in ./packages/*/;
+          do
+            echo "Installing dependencies for $package"
+            cd $package && npm install && cd ../../
+          done
```

Comment:
The lerna docs mentions [switching](https://lerna.js.org/docs/legacy-package-management#replacing-your-usage-of-lerna-bootstraplerna-link) from using bootstrap to using [npm workspaces](https://docs.npmjs.com/cli/v10/using-npm/workspaces). Is there a reason we didnt want to utilize workspaces here?